example/src/pages/Login.tsx

```diff
@@ -1,20 +1,32 @@
-import styled, { css } from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
 import React, { useState, useCallback, ChangeEvent } from 'react';
-import { ButtonWithLoading, TextField, PasswordField, Form, AlertBar } from 'scorer-ui-kit';
+import { ButtonWithLoading, TextField, PasswordField, Form, AlertBar, AlertWrapper } from 'scorer-ui-kit';
 import GhostLogo from '../svg/ghost-logo.svg';
 import {LoginScreen} from '../svg';
+import {Link} from 'react-router-dom';
 
 const widthDesk = 480;
 
+const gradients = {
+  "primary" : `linear-gradient(-45deg, #5CA0D1, #7DB8DB)`,
+  "secondary" : `linear-gradient(139deg, hsl(250, 60%, 62%), hsl(0, 46%, 54%))`,
```

Comment:
Minor: can we use consistent units here? also why are these template strings?

Reply:
Hi!, yes sorry I left the values as they were, will update with the hsl ones.

Reply:
Also to answer the template strings was added not to prevent any error, just thought it would be natural to use it like that since they are used in the styled component values, but I can update to normal strings.

Reply:
It's better to stick with using hsl / hsla for colours as they're much more readable and easy to work with than hex of rgb.